idoit-install

```diff
@@ -1793,7 +1793,27 @@ function installIDoit {
     log "Install i-doit via console.php"
     echo -n -e \
         "Please enter a Admin Center password [leave empty for '${IDOIT_ADMIN_CENTER_PASSWORD}']: "
-    read -r answer
+    read -r AdminCenterPass
```

Comment:
Please start local variables with non-capitalized characters

Reply:
Changed, thanks for the Hint!